### ISSUE TITLE
[BUG] Reveal opened file in coc-explorer

### ISSUE THREAD

**Issue by Flamme13** [weirongxu/coc-explorer#547]
Today after the update I have a problem focusing on a file in explorer after opening it.

Message:
"explorer.file.autoReveal has been deprecated, please use explorer.file.reveal.whenOpen instead of it"

My coc-settings.json for coc-explorer looks like this:
```
  "explorer.position": "right",
  "explorer.width": 50,
  "explorer.quitOnOpen": true,
  "explorer.previewAction.onHover": "content",
  "explorer.previewAction.content.maxHeight": 90,
  "explorer.icon.enableNerdfont": true,
  "explorer.file.hiddenRules": {
    "filenames": [".git"],
    "patternMatches": []
  },
```

**Info**

vim version: NVIM v0.7.0
node version: v16.16.0
coc.nvim version: 0.0.82-a3d271d8 2022-10-27 16:26:01 +0800
coc.nvim directory: /home/flamme/.vim/plugged/coc.nvim
term: tmux
platform: linux

**Steps to reproduce the behavior**

1. Open some file
2. Open coc-explorer (`nmap <space>e <Cmd>CocCommand explorer<CR>`)
3. See a problem

**Expected behavior**

After opening coc-explorer in the current buffer focusing this file in coc-explorer

**Comment by ElcomJ** [comment]
Good morning, mine is also experiencing the same problem.

As soon as I run the `:CocCommand explorer` command, this error appears.

Image of the error:

![image](https://user-images.githubusercontent.com/37147897/198271642-24d31f18-7288-4d77-918c-652528a3f696.png)

**Comment by ElcomJ** [comment]
Hey @weirongxu, the problem continues even after the commit 2275f68.

PS: I have already run `:CocUpdate` in order to receive the updates from the commit 2275f68.

**Comment by weirongxu** [comment]
Thanks for testing and following up, should be fixed in version 0.25.3

**Comment by ElcomJ** [comment]
Thank you, I ran `:CocUpdate` again and as you mentioned, the problem was fixed in the commit f7bbd25

Thanks for your help and have a great day :)
